rust_src/src/dired.rs

```diff
@@ -1,16 +1,74 @@
 //! Lisp functions for making directory listings.
 
-#[cfg(not(windows))]
-use libc;
+#[cfg(unix)]
+include!("dired_unix.rs");
```

Comment:
My other question is about these includes. Was there no way to use modules? I suspect it would be nicer.

Reply:
Good pt -- just did not think of it -- switching to modules now.